velox/buffer/Buffer.h

```diff
@@ -686,6 +686,13 @@ class BufferView : public Buffer {
     return result;
   }
 
+  // Helper method to create a buffer view referencing another existing Buffer.
+  static BufferPtr
+  create(BufferPtr innerBuffer, Releaser releaser, bool podType = true) {
```

Comment:
In the case `releaser` is the `innerBuffer` itself, there is no need to pass it separately
```c++
  static BufferPtr
  create(const BufferPtr& innerBuffer, bool podType = true) {
    return create(innerBuffer->as<uint8_t>(), innerBuffer->size(), innerBuffer, podType);
  }
```

Reply:
@Yuhta innerBuffer is not necessarily a `BufferView`. It could be just a regular `Buffer`, and in that case it doesn't contain a `releaser`.